src/main/kotlin/no/skatteetaten/aurora/boober/service/AuroraDeploymentContextService.kt

```diff
@@ -288,6 +289,14 @@ class AuroraDeploymentContextService(
             "Both sts and certificate feature has generated a cert. Turn off certificate if you are using the new STS service"
         } else null
 
-        return listOfNotNull(websealWarning, stsWarning).addIfNotNull(configKeysWithSpecialCharacters)
+        val alertsConnection = features.filter { (feature, spec) ->
```

Comment:
Når jeg ser dette så tenker jeg at vi bør vurdere å flytte slike funksjoner ut i en spec fil. For eksempel at en fil som het AlertsSpec.kt, som hadde en funksjon:
```kotlin
fun AuroraDeploymentSpec.containsDeprecatedAlertConnection() {}
```
Da hadde vi slippet å filtrere på features og sjekket at den var AlertsFeature. Hvis dette hadde vært mulig så kunne hele metoden findWarnings vært betydelig enklere. 
Kanskje vi bør ha en oppgave på det etterhvert. 
Du trenger ikke gjore noe med det nå. Bare en tanke

Reply:
Jeg synes det høres lurt ut, og kan foreslås.
Det kunne også vært en spec interface/abstract med containsDeprecatedConfiguration, som hver feature kunne override.
Alternativ kunne det være en funksjon på spec som returnerer advarsel som er spesifikk for featuren, så slipper vi at det blør feature-spesifikke warnings i findWarnings-funksjonen.

Reply:
Det høres heller ikke dumt ut. Vi kan tenke litt på det. Warnings er ikke helt 100% bra slik det er i dag. Det skulle vel ideelt sett ha vært noe slik at man kunne satt det for AuroraConfigField, det hadde vært veldig greit i dette tilfellet, men det er nok en større jobb. 

Det hadde gjort det enklere å deprekere ting. Man kunne også sett for seg at man kunne sendt usage over slike felter til splunk eller som metrikker. Da kunne man visst når man kan skru av deprecated felter f.eks